ssh/DOCS.md

```diff
@@ -1,55 +1,63 @@
-# Home Assistant Add-on: SSH server
+# Home Assistant Add-on: SSH Server
```

Comment:
Let's correct the name so it matches the actual add-on name, see:

<https://github.com/home-assistant/hassio-addons/blob/master/ssh/config.json#L2>

```suggestion
# Home Assistant Add-on: Terminal & SSH
```